static/app/components/feedback/feedbackItem/feedbackReplay.tsx

```diff
@@ -21,7 +21,10 @@ export default function FeedbackReplay({eventData, feedbackItem, organization}:
   const {feedbackHasReplay} = useReplayCountForFeedbacks();
   const hasReplayId = feedbackHasReplay(feedbackItem.id);
 
-  const replayId = eventData?.contexts?.feedback?.replay_id;
+  // replay ID can be found in two places
+  const replayId =
+    eventData?.contexts?.feedback?.replay_id ??
+    eventData?.tags?.find(({key}) => key === 'replayId')?.value;
```

Comment:
similar but different to https://github.com/getsentry/sentry/blob/d647fbb02240343f843c1a6d90dda749e39a43f0/static/app/utils/replays/getReplayIdFromEvent.tsx#L3-L8